dspace-api/src/test/data/dspaceFolder/assetstore/scielo-test.txt

```diff
@@ -0,0 +1,51 @@
+TY  - JOUR
```

Comment:
Again, this file should move to `dspace-server-webapp/src/test/resources/org/dspace/external/provider/impl/`